File: VB/WebSite/clsDetail.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls

''' <summary>
''' Summary description for clsDetail
''' </summary>
Public Class clsDetail
	Private _code As String = ""
	Private _gender As String = ""
	Private _titleOfCourt As String = ""
	Private _address As String = ""

	Public Sub New(ByVal code As String, ByVal gender As String, ByVal titleOfCourt As String, ByVal address As String)
		_code = code
		_gender = gender
		_titleOfCourt = titleOfCourt
		_address = address
	End Sub

	Public ReadOnly Property Code() As String
		Get
			Return _code
		End Get
	End Property

	Public ReadOnly Property Gender() As String
		Get
			Return _gender
		End Get
	End Property

	Public ReadOnly Property TitleOfCourt() As String
		Get
			Return _titleOfCourt
		End Get
	End Property
	Public ReadOnly Property Address() As String
		Get
			Return _address
		End Get
	End Property
End Class

File: CS/WebSite/clsDetail.cs
using System;
using System.Data;
using System.Configuration;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;

/// <summary>
/// Summary description for clsDetail
/// </summary>
public class clsDetail
{
    private string _code = "";
    private string _gender = "";
    private string _titleOfCourt = "";
    private string _address = "";

    public clsDetail(string code, string gender, string titleOfCourt, string address)
    {
        _code = code;
        _gender = gender;
        _titleOfCourt = titleOfCourt;
        _address = address;
    }

    public string Code
    {
        get { return _code; }
    }

    public string Gender
    {
        get { return _gender; }
    }

    public string TitleOfCourt
    {
        get { return _titleOfCourt; }
    }
    public string Address
    {
        get { return _address; }
    }
}
